### ISSUE TITLE
Analyze ROI data not working for multiple videos

### ISSUE THREAD

**Issue by breannashi** [sgoldenlab/simba#363]
Here are the steps I took:
1.) added video and DLC csv to existing project
2.) added a measurement for the new video in video parameters 
3.) Defined ROI for the new video 
4.) run analyze roi data aggregates and Analyze Distances 
Problem: 
for step 4 I am only getting results for the first video, when I want results for the new video. 
Video of what I am seeing on my end found here:
https://youtu.be/yxb-5UfGMjw

Thank you!

**Comment by sronilsson** [comment]
Hi @breannashi - I think (maybe wrong) someone reported this issue a few weeks ago and we fixed it. 

Which version of SimBA are you running? `pip show simba-uw-tf-dev`

If you run `pip install simba-uw-tf-dev --upgrade` do you still see this error?

PS. I think github has fixed so you can drag and drop video mp4 files straight into these chats.

**Comment by sronilsson** [comment]
Ah I can see the version number in your video :) Anyway, try to upgrade to latest as above, and let me know if that fixes it or not please.

**Comment by breannashi** [comment]
This did not fix the error. It is functioning exactly the same.

**Comment by breannashi** [comment]
for context:
Simba: SimBA v.1.94.2
OS: Ubuntu 22.04.4 LTS

**Comment by sronilsson** [comment]
Thanks @breannashi - very helpful. So I better understand the issue, please correct me if I am wrong:

You first define ROIs on more than one videos, you then analyze ROI aggregate data including distances moved for these videos. However, the results you see in only contain data for a single video? 

Sorry I only have a Mac at present, but are you essentially doing this process in the video below, but the final CSV only has data for one video?

https://github.com/sgoldenlab/simba/assets/34761092/f1a76cfc-8739-4df8-8923-1bca9c1c4342

**Comment by breannashi** [comment]
Yes that's right. and where your simba prints that it runs on both videos mine only prints one.

**Comment by sronilsson** [comment]
Thanks @breannashi, two more questions:

1) Can you send me your `/project_folder/logs/measures/ROI_definitions.h5` file and I will take a look at it? This file stores your drawn ROI definitions. You should be able to zip it and drop it into this chat thread.

2) How many files do you have represented in the `/project_folder/csv/outlier_corrected_movement_location` directory of your SimBA project - can you send me a screenshot of the content of this directory?

**Comment by breannashi** [comment]
Thank you! here are the files requested!
![image](https://github.com/sgoldenlab/simba/assets/74272793/7e05c50b-3ac6-4990-8502-56dfad8fe9f1)
![image](https://github.com/sgoldenlab/simba/assets/74272793/54f098d3-9f12-47cd-b30a-b75bc3d9f0d2)
[ROI_definitions.zip](https://github.com/sgoldenlab/simba/files/15487018/ROI_definitions.zip)

**Comment by sronilsson** [comment]
Thanks @breannashi!

For the screenshoot, I can see that you have data for a single video (`CC1_tank_exploration_29-11-22`). 

Opening the ROI definitions, I can also see that you have polygons drawn for two videos (with or without the `_29-11-22` suffix):

<img width="489" alt="image" src="https://github.com/sgoldenlab/simba/assets/34761092/7019aa4a-ca76-4bea-91d7-4f29936b493b">

If you want to compute the ROI data for both `CC1_tank_exploration_29-11-22` and `CC2_Tank_exploration`, there needs to be data-files inside the `/project_folder/csv/outlier_corrected_movement_location` directory representing the two videos. 

The reason you are getting results for only a single video file, is likely because there is only a single file in this directory, SimBA can't fond any more data.

**Comment by breannashi** [comment]
That makes sense. I am wondering why it defaults to add my regions to video 1 csv rather than make a new one? and will the .h5 also need to be altered?

**Comment by sronilsson** [comment]
Hi @breannashi - I am note completly sure what you mean. When you say "defaults" what do you mean? 

Some notes that may be relevant: 

The videos you see listed in the image below are videos that SimBA find in the `project_folder/videos/` directory of your SimBA project. If the video is not there, there is nothing to draw your regions on. Add the video to that folder to be able to draw on it.

<img width="990" alt="image" src="https://github.com/sgoldenlab/simba/assets/34761092/715a292e-423c-49c1-a559-b957def39544">

To duplicate the ROIs you've drawn on one video, to all the other videos listed, click the `Apply to all` button. For example, of I have drawn polygons on the first video in the list, but not the others, but I want the same polygons on the remaining 7 videos, I click the `apply to all` button associated with row 1: 

<img width="566" alt="image" src="https://github.com/sgoldenlab/simba/assets/34761092/05521c1f-14f5-4e0b-b4b5-589a5021c100">

**Comment by breannashi** [comment]
Never-mind! I did not read properly. The problem is I forgot to click skip outlier correction. Thank you for your help!!

**Comment by sronilsson** [comment]
No problem, just let me know if anything else comes up
